examples/nlp/language_modeling/conf/megatron_gpt_prompt_learning_config.yaml

```diff
@@ -57,6 +57,8 @@ model:
   validation_global_batch_size: ${model.global_batch_size}
   validation_micro_batch_size: ${model.micro_batch_size}
   validation_drop_last: False
+  report_validation_metric: False
+  validation_metric: 'bleu'
```

Comment:
Is there a reason why the default validation metric on gpt and t5 are not the same?